completions/_git-forgit

```diff
@@ -99,3 +97,30 @@ _git-forgit() {
 		;;
 	esac
 }
+
+# Check if forgit plugin is loaded
+if (( $+functions[forgit::add] )); then
+	# We're reusing existing completion functions, so load those first
+	# if not already loaded and check if completion function exists afterwards.
+	(( $+functions[_git-add] )) || _git
+	(( $+functions[_git-add] )) || return 1
+	(( $+functions[_git-branches] )) || _git-forgit
+	(( $+functions[_git-branches] )) || return 1
```

Comment:
These two lines are not necessary anymore, when this file is being executed the forgit completion functions have obviously been loaded.